subsys/bluetooth/shell/audio.c

```diff
@@ -54,75 +57,75 @@ struct named_lc3_preset {
 };
 
 static struct named_lc3_preset lc3_unicast_presets[] = {
-	{"8_1_1",   BT_AUDIO_LC3_UNICAST_PRESET_8_1_1},
-	{"8_2_1",   BT_AUDIO_LC3_UNICAST_PRESET_8_2_1},
-	{"16_1_1",  BT_AUDIO_LC3_UNICAST_PRESET_16_1_1},
-	{"16_2_1",  BT_AUDIO_LC3_UNICAST_PRESET_16_2_1},
-	{"24_1_1",  BT_AUDIO_LC3_UNICAST_PRESET_24_1_1},
-	{"24_2_1",  BT_AUDIO_LC3_UNICAST_PRESET_24_2_1},
-	{"32_1_1",  BT_AUDIO_LC3_UNICAST_PRESET_32_1_1},
-	{"32_2_1",  BT_AUDIO_LC3_UNICAST_PRESET_32_2_1},
-	{"441_1_1", BT_AUDIO_LC3_UNICAST_PRESET_441_1_1},
-	{"441_2_1", BT_AUDIO_LC3_UNICAST_PRESET_441_2_1},
-	{"48_1_1",  BT_AUDIO_LC3_UNICAST_PRESET_48_1_1},
-	{"48_2_1",  BT_AUDIO_LC3_UNICAST_PRESET_48_2_1},
-	{"48_3_1",  BT_AUDIO_LC3_UNICAST_PRESET_48_3_1},
-	{"48_4_1",  BT_AUDIO_LC3_UNICAST_PRESET_48_4_1},
-	{"48_5_1",  BT_AUDIO_LC3_UNICAST_PRESET_48_5_1},
-	{"48_6_1",  BT_AUDIO_LC3_UNICAST_PRESET_48_6_1},
+	{"8_1_1",   BT_AUDIO_LC3_UNICAST_PRESET_8_1_1(LOCATION, CONTEXT)},
```

Comment:
I'm not sure it makes sense to have a compile-time defined location. When setting up a stereo stream you typically want to make two different locations - RIGHT to one stream and LEFT to the other - or pack left and right into the same stream.

Reply:
Agreed :)